README.md

```diff
@@ -0,0 +1,44 @@
+# Repository for the open-source pinball game red-roses

```

Comment:
It's not a pinball game. Actually this one is heavily inspired by the "warlock"-mod from WC3